driver-core/src/main/com/mongodb/operation/AggregateExplainOperation.java

```diff
@@ -51,14 +51,20 @@
 class AggregateExplainOperation implements AsyncReadOperation<BsonDocument>, ReadOperation<BsonDocument> {
     private final MongoNamespace namespace;
     private final List<BsonDocument> pipeline;
+    private final boolean retryReads;
     private Boolean allowDiskUse;
     private long maxTimeMS;
     private Collation collation;
     private BsonValue hint;
 
     AggregateExplainOperation(final MongoNamespace namespace, final List<BsonDocument> pipeline) {
+        this(namespace, pipeline, true);
+    }
+
+    AggregateExplainOperation(final MongoNamespace namespace, final List<BsonDocument> pipeline, final boolean retryReads) {
```

Comment:
For all the operations, make retryReads a property with its own fluent setter, rather than overload all the constructors.  We generally reserve constructor parameters for properties that are required for execution, and not ones with sensible defaults. That avoids constructor churn in all the operations as we add new properties over time

Reply:
Done. I have also done this for the associated `Iterable` classes, as well as for `Operations`, `SyncOperations` and `AsyncOperations`.

Reply:
See my comment above about when to use fluent setters and when to use constructor parameters

Reply:
Done.